lib/site_prism/section.rb

```diff
@@ -31,23 +30,40 @@ def initialize(parent, root_element, &block)
       within(&block) if block_given?
     end
 
+    ::Capybara::Session::NODE_METHODS.each do |method|
+      class_eval <<~METHOD, __FILE__, __LINE__ + 1
```

Comment:
I had a look into this on ruby docs but couldn't find any good examples of what it does / how it works. Do you have any good links for me to read up on?